core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/cluster/ClusterV4RequestToClusterConverter.java

```diff
@@ -66,7 +66,7 @@
     @Value("${cb.ambari.username:cloudbreak}")
     private String ambariUserName;
 
-    @Value("${cb.ambari.dp.username:dpapps}")
```

Comment:
This looks a bit wierd: a CM variable under ambari and dp.

Reply:
yep, same as above, I don't want to change these now

Reply:
actually I could rename the env vars, because we're not overriding them anywhere, but in the DB they would be under the same ambari columns

Reply:
I will change this one